source/blog/2016-10-04-ovirt-provider-ovn.md

```diff
@@ -0,0 +1,160 @@
+This can be done manually or by adding the ovirt-provider-ovn firewalld service to the appropriate firewalld zone.
+
+    firewall-cmd --zone=<zone to add service to> --add-service=ovirt-provider-ovn --permanent
+    firewall-cmd --reload
+
+The zone currently selected as default can be obtained by executing:
+
+    firewall-cmd --get-default-zone
+
+After installation, the provider can be started as follows:
+
+    systemctl start ovirt-provider-ovn
+
+The provider can then be added to oVirt as an external network provider. In order to add a new provider, go to the External Providers section in the oVirt UI and click the Add button.
+
+![adding a new provider](uarwo40-edit-cluster.png)
+
+When the provider is successfully added, existing OVN networks can be imported to oVirt.
+New OVN networks can be defined using oVirt by adding a network and specifying it to be added on an external provider (make sure you add the external provider in read-write mode, so that you can add external networks from oVirt).
+
+A VM NIC can be added to OVN networks by simply choosing an external network during NIC provisioning.
+
+OVN based networking brings many advantages to oVirt:
+* More granular security - it brings complete network isolation without the need for defining VLANs.
+* Easier management - the management of the networking infrastructure should also become much easier, as instead of managing the network isolation on each of the networking components, it will be automatically taken care of by OVN.
+
+## Building oVirt OVN Provider rpms From Repository
+
+Clone the repository:
+
+    git clone https://github.com/openvswitch/ovirt-provider-ovn`
+
+Build the rpms:
+
+    cd ovirt-provider-ovn
+    make rpm
+
+The built rpms can be found in: `~/rpmbuild/RPMS/noarch/`
+
+## Building OVN Provider rpms From Repository
+
+Clone the repository:
+
+    git clone https://github.com/openvswitch/ovs
+
+Install the following packages, as they are need to build ovn:
+
+    yum -y install gcc make python-devel openssl-devel kernel-devel graphviz kernel-debug-devel autoconf automake rpm-build redhat-rpm-config rpm-build rpmdevtools bash-completion autoconf automake libtool PyQt4 groff libcap-ng-devel python-twisted-core python-zope-interface graphviz openssl-devel selinux-policy-devel
+
+Build the ovn rpms:
+
+    cd ovs
+    ./boot.sh
+    ./configure
+    make dist
+    cp openvswitch-<version>.tar.gz $HOME/rpmbuild/SOURCES
+    cd $HOME/rpmbuild/SOURCES
+    tar xzf openvswitch-<version>.tar.gz
+    cd openvswitch-<version>
+    rpmbuild -bb rhel/openvswitch-fedora.spec
+
+The built rpms will reside here: `~/rpmbuild/RPMS/x86_64/`
+
+## Tested environments
+
+This guide has been tested on both Fedora24 and RHEL7 environments.
+
+Note: Currently a OVS spec file for Fedora is the only available in the OVS repo. We are however using the generated rpm's on RHEL7 and have not encountered any problems so far. Please consult the OVS team for more info.
```

Comment:
I was more worried about our own spec file and rpm. but I can live with current text.